docs/log-collector/README.md

```diff
@@ -43,6 +45,34 @@ tvk-log-collector collects the logs, config and events of resources. Pod Logs ca
   kubectl krew uninstall tvk-log-collector
   ```
 
+#### 2. Without `krew`:
+
+1. List of available releases: https://github.com/trilioData/tvk-plugins/releases
+2. Choose a version of log-collector plugin to install and check if release assets have log-collector plugin's package
+   [log-collector_${version}_${OS}_${ARCH}.tar.gz] for your desired OS & Architecture.
+3. Set env variable `version=v1.x.x` [update with your desired version].
+
+##### Linux/macOS
+
+- Bash or ZSH shells
+```bash
+(
+  set -ex; cd "$(mktemp -d)" &&
+  OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+  ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+  package_name="log-collector_${version}_${OS}_${ARCH}.tar.gz" &&
+  curl -fsSLO "https://github.com/trilioData/tvk-plugins/releases/download/"${version}"/${package_name}" &&
+  tar zxvf ${package_name} && sudo mv log-collector /usr/local/bin/kubectl-tvk_log_collector
+)
+```
+Verify installation with `kubectl tvk-log-collector --help`
+
+##### Windows
+
+1. Download `log-collector_${version}_windows_${ARCH}.zip` from the Releases page to a directory and unzip the package.
```

Comment:
Also help for getting ARCH on windows. I think user will have to use systeminfo instead of uname on windows